### ISSUE TITLE
Title and shortname for our guidance

### ISSUE THREAD

**Issue by JJdeGroot** [w3c/matf#76]
We have to choose a definitive title a shortname for our guidance.

They currently are:

Title: `Applying WCAG 2.2 to Mobile`
Shortname: `mobile-accessibility-mapping-22`

This is how it will ook in the published note:

<img width="831" alt="Screenshot of mobile guidance using respec" src="https://github.com/user-attachments/assets/5bf68a87-e82e-4fa4-8e57-2f73515175d8">

We can choose a title and shortname to our liking.

[WCAG2ICT](https://www.w3.org/TR/wcag2ict-22/):
- Title: `Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)`
- Shortname: `wcag2ict-22`

[WCAG](https://www.w3.org/TR/WCAG22/):
- Title: Web Content Accessibility Guidelines (WCAG) 2.2
- Shortname: `WCAG22`

**Comment by JJdeGroot** [comment]
Considering we want to align with WCAG2ICT where possible, my proposal is:

- Title: `Guidance on Applying WCAG 2.2 to Mobile (WCAG2Mobile)`
- Shortname: `wcag2mobile-22`

I think WCAG2Mobile is an easy to remember name. I would add `2.2` (instead of 2) in the title to emphasize our guidance is recent. I think Mobile should be capitalized, not uppercased, because ICT is an abbreviation and Mobile is not.

Add your proposal below, or thumbs up if you agree.

**Comment by jamieherrera** [comment]
I'd drop the 22...

**Comment by JJdeGroot** [comment]
Discussed in today’s meeting.

<details>
<summary>6 November 2024</summary>

[Source](https://www.w3.org/2024/11/06-matf-minutes.html#eeea)

~~~text
JJ Name is not very consistent, suggesting Guidance on Applying WCAG 2.2 to Mobile (WCAG2Mobile) and Shortname: wcag2mobile-22

<JJ> Link to guidance naming: [w3c/matf#76](https://github.com/w3c/matf/issues/76)

Jamie So this isn't a continuation of the prior draft note?

JJ no

JJ will remove references to 2.0

JJwe should point to latest doc

Joe_Humbert Is there a way to let people know what's outdated?

JJ there is an archived repo (Mobile-A11y-TF-Note) that we can update

ACTION: Update W3C Mobile Accessibility Page

ACTION: Check if we can update published (outdated) guidance with link to new guidance
~~~
</details>

ACTION: Update W3C Mobile Accessibility Page

ACTION: Check if we can update published (outdated) guidance with link to new guidance

**Comment by detlevhfischer** [comment]
I agree "Applying WCAG 2.2 to Mobile" sounds nice and easy to remember BUT - this point has been made repeatedly in AGWG:  "mobile" is a misnomer. 
* You have websites on mobile devices
* Tablets with keyboard and mobile OSs and laptops with Desktop OSs with touch mean that the distinction becomes blurred
* The key thing would be **native** apps on mobile OSs (currently iOS or Android, discounting skin variants on the latter).

So I think this should be discussed in the main working group.

**Comment by patrickhlauke** [comment]
yes, dear god yes. "mobile" is a meaningless term in and of itself. it's ambiguous, and doesn't actually get to the core of what you're trying to address. i'd strongly oppose this idea, i'm afraid

**Comment by JJdeGroot** [comment]
Hi @detlevhfischer and @patrickhlauke , thanks for raising your (repeated) concerns with the term "mobile".

From a web perspective, I agree that the term doesn't cover what we are trying to address.

I also agree that our Task Force should focus on native mobile apps, because that's where the gaps are. 

Per our Work Statement, our objective is: "The objective of the Task Force is to produce informative resources for applying WCAG to Mobile Applications, including but not limited to native mobile apps, mobile web apps, mobile web content, and hybrid apps using web components inside native mobile apps."

Today, in 2024, I think there is little reason to consider mobile web accessibility separately from web accessibility; especially since the AG WG already takes mobile into account.

However, from an app perspective, my experience is that developers consider themselves 'mobile developers' and use 'mobile accessibility' to refer to the accessibility of native mobile applications; namely Android and iOS.

What do you think about the theoretical situation where we rename ourselves to App Accessibility Task Force, remove mobile web content from our objective, and name our guidance WCAG2App?

**Comment by jha11y** [comment]
WCAG2MobileApp?

**Comment by JJdeGroot** [comment]
Discussed in today’s meeting.

<details>
<summary>13 November 2024</summary>

[Source](https://www.w3.org/2024/11/13-matf-minutes.html#2c88)

~~~text
There are multiple suggestion on github. Mobile is not preffered.

*the term mobile in the guidance name is not preffered

because it includes and not limited to native mobile apps, mobile web apps, mobile web content, and hybrid apps using web components inside native mobile apps.

<Joe_Humbert> +1 for now, though I feel others like Patrick will push back

<JJ> Do we agree to use the title "Guidance on Applying WCAG 2.2 to Mobile (WCAG2Mobile)" for our draft, keeping the option open to change the term "Mobile" in a future stage?

<Aash> +1 to JJ

Guidance on Applying WCAG 2.2 to Mobile (WCAG2Mobile) can be used as a working title

later we can drop the term "mobile"

<Illai> +1

<Karla> I prefer WCAG22Mobile

<Tim> +1 to Guidance on Applying WCAG 2.2 to Mobile (WCAG2Mobile)

Karla should it not be WCAG22-2-Mobile?

<Karla> Yes, Guidance on Applying WCAG 2.2 to Mobile

ACTION: Keep "Mobile" definition as is, keep option open to change it in the future
~~~
</details>

Conclusion: we are going to keep the Mobile definition for our first draft. In future versions, we are keeping the option open to choose a term which is less ambiguous. But this also depends on the boundaries that we have within W3C.

- Title: `Guidance on Applying WCAG 2.2 to Mobile (WCAG2Mobile)`
- Shortname: `wcag2mobile`

**Comment by jamieherrera** [comment]
In the abstract or other area of the document we can and should define our use and reasoning for "mobile" as outlined above, including challenges to the term. Noted.

**Comment by detlevhfischer** [comment]
I would have preferred @jha11y s suggestion "WCAG2NativeMobileApps" but I'm certainly not going to kick up a fuss about the naming of the thing (couldn't take part in today's meeting).

**Comment by shawna-slh** [comment]
The previous document title was frontloaded with 'Mobile Accessibility':
> Mobile Accessibility: How WCAG 2.2 and Other W3C/WAI Guidelines Apply to Mobile

This proposed title is similar to the [WCAG2ICT doc](https://www.w3.org/WAI/standards-guidelines/wcag/non-web-ict/)
> Guidance on Applying WCAG 2.2 to Mobile (WCAG2Mobile)

I think there is value in front-loading with 'Mobile Accessibility' for SEO and for [messaging](https://www.w3.org/WAI/standards-guidelines/mobile/) "Mobile accessibility is covered in existing W3C WAI accessibility standards/guidelines. There are not separate guidelines for mobile accessibility."

Also, I find 'WCAG2Mobile' a bit hard to read. I wonder if hyphens would be better for the title? (WCAG-2-Mobile) (You wouldn't need them for the shortname.) Or, actually, you don't even need 'WCAG2Mobile' in the title at all.

> Shortname: wcag2mobile

Therefore, please consider title:
> Mobile Accessibility: Guidance on Applying WCAG 2.2 to Mobile (WCAG-2-Mobile)

or
> Mobile Accessibility: Guidance on Applying WCAG 2.2 to Mobile

**Comment by JJdeGroot** [comment]
Hi @shawna-slh , I've brought up this issue with our core editor's team.

It's correct that "mobile accessibility" is covered in existing W3C WAI accessibility standards/guidelines. E.g. a lot of mobile concepts have been added in WCAG 2.1 and 2.2. But "mobile" remains an ambiguous term. Everything that's "mobile web" is covered in existing standards/guidelines. But other things mobile, such as "native mobile applications", are not sufficiently covered. Hybrid applications (native mobile applications with web components) are also not covered. Mobile web applications are somewhat covered, but could use more guidance.

That's why our Abstract is:

> This document, “Guidance on Applying WCAG 2.2 to Mobile (WCAG2Mobile)”, describes how the Web Content Accessibility Guidelines (WCAG) version 2.2 [[WCAG22](https://www.w3.org/TR/WCAG22/)] principles, guidelines, and success criteria can be applied to **Mobile Applications, including but not limited to native mobile apps, mobile web apps and hybrid apps using web components inside native mobile apps.** It provides informative guidance (guidance that is not normative and does not set requirements).

As proposed by some participants, the most obvious clarification is changing "Mobile" to "Mobile Applications".

Compared to the [2015 note](https://www.w3.org/TR/mobile-accessibility-mapping/), we have dropped "mobile web content" from our scope, because that's fully covered in other standards/guidelines, e.g. when mobile web content is rendered using a mobile web browser.

Now that "mobile web content" is no longer in scope, the remaining three focus points are all "mobile applications". 

How about:

>  Guidance on Applying WCAG 2.2 to Mobile Applications

or

> Guidance on Applying WCAG 2.2 to Mobile Applications (WCAG2APP)

or

> Mobile Accessibility: Guidance on Applying WCAG 2.2 to Mobile Applications (WCAG2APP)

As for the shortname/slug:

- wcag2app
- wcag2mobile
- wcag2ma
- matf

are some of the options?

My personal opinion is that we should not prefix mobile accessibility, given that it's an ambiguous term. I also think it's good to provide a shortname in the title like WCAG2ICT, because otherwise readers might abbreviate it differently.

**Comment by JJdeGroot** [comment]
Discussed in today’s meeting. (@shawna-slh)

<details>
<summary>5 March 2025</summary>

[Source](https://www.w3.org/2025/03/05-matf-minutes.html#0426)

~~~text
<JJ> [w3c/matf#76](https://github.com/w3c/matf/issues/76)

JJ WCAG2Mobile title update: Had a meeting and pre cfc and they would prefer we use this as the title. Plus they think it will help SEO and local web site search functions. There was some discussion regarding to what mobile means, running on a browser / tablet / wearables. Currently the guidance says that the WCAG is applied to applications running

on mobile. JJ suggests that we change this to Mobile Applications. Shaun re-opened an issue. JJ: Should we prefix mobile accessibility? Should we make it WCAG2APP? Mobile being at the start and end makes it more clear for the W3C.

<pauljadam> Mobile Accessibility: Guidance on Applying WCAG 2.2 to Mobile Apps

<pauljadam> Mobile Accessibility: Guidance on Applying WCAG 2.2 to Native Apps

<JJ> Suggestion: Mobile Accessibility: Guidance on Applying WCAG 2.2 to Mobile Applications

<pauljadam> Just not sure we want to say Mobile twice in the title?

<JJ> https://www.w3.org/WAI/standards-guidelines/mobile/

Jamie earlier you mention a W3C page that we should align with - which one are they referring to? (@JJ) - I just want to recap what's on there. At one point it was talking about mobile web stuff

<Aash> Mobile Accessibility: Guidance on Applying WCAG 2.2 to portable devices - WCAG2Portable

<Aash> Mobile Accessibility: Guidance on Applying WCAG 2.2 to devices on the go - WCAG2OntheGo

JJ an old document prefixes with mobile. Mobile accessibility is so broad.

julianmka it does distinguish between mobile web and mobile apps. If they want us to change our name they should change their page

julianmka W3C mobile pages look like they need updating

<JJ> https://www.w3.org/WAI/standards-guidelines/wcag/non-web-ict/ is subpage of WCAG, mobile accessibility is direct subpage of Standards/Guidelines

<Jamie> quintinb the above comments are mine not julianmka :P

Sorry

<pauljadam> +1 on adding native in the title

That was a very quick discussion

<pauljadam> +1 on not having mobile in there twice

RobW I want to ask about the "native" aspect - it depends on our remit.

<Aash> chatGPT suggests this one: WCAG2ONE (Optimized for Native Experiences)

<Aash> Mobile Accessibility: Guidance on Applying WCAG 2.2 to Optimized for Native Experiences

<Jamie> +1 to Suggestion: Guidance on Applying WCAG 2.2 to Mobile Applications (WCAG2

JJ have a vote for 3 options: combinations with prefix and suffix

<Jamie> mobile)

<Jamie> what about WCAG2Mobile

JJ Mobile Applications is the furthest we can stretch it

<pauljadam> Apps seems more native than Applications

<pauljadam> There is pure native and there is hybrid

@quintin Our definition needs to be clear that native is not html (e.g.

<pauljadam> Yes it's more formal

<JJ> 1: Mobile Accessibility: Guidance on Applying WCAG 2.2 to Mobile Applications

<JJ> 2: Guidance on Applying WCAG 2.2 to Mobile Applications

<JJ> 3: Guidance on Applying WCAG 2.2 to Mobile

<JJ> Please type 1, 2, 3 to vote on your preferred title

1

<Joe_Humbert> 2

<pauljadam> 2

<jeroen> 1

<Carolina> 2

<Illai> 2

<julianmka> 2

<RobW> 2

<RacheleD> 2

<Jamie> 2

<Aash> 1

Joe_Humbert Reiterating that we don't like the repeat of the word mobile

ACTION: Group voted for title: "Guidance on Applying WCAG 2.2 to Mobile Applications (WCAG2Mobile)"

Jamie As a comprise - we should ask them to update the Mobile A11y page, and put our page in the same context. Give us a reason why the page is separate from WCAG2ICT

ACTION: Check why Mobile Accessibility Note is not a subpage of WCAG page

Jamie Mobile Accessibility needs to mean "Mobile Applications"
~~~
</details>

**POLL**
- Option 1 has 3 votes: `Mobile Accessibility: Guidance on Applying WCAG 2.2 to Mobile Applications (WCAG2Mobile)`
- Option 2 has 8 votes: `Guidance on Applying WCAG 2.2 to Mobile Applications (WCAG2Mobile)`
- Option 3 has 0 votes: `Guidance on Applying WCAG 2.2 to Mobile (WCAG2Mobile)` (current)

**ACTIONS**
- ACTION: Group voted for title: "Guidance on Applying WCAG 2.2 to Mobile Applications (WCAG2Mobile)"
- ACTION: Check why Mobile Accessibility Note is not a subpage of WCAG page

**Comment by shawna-slh** [comment]
> "Guidance on Applying WCAG 2.2 to Mobile Applications (WCAG2Mobile)"

I was not aware of some of the background from the discussion. I'll check in with @iadawn and AG Chairs

> ACTION: Check why Mobile Accessibility Note is not a subpage of WCAG page

Do you mean [Mobile Accessibility at W3C](https://www.w3.org/WAI/standards-guidelines/mobile/)? That page covers more than WCAG. We can revisit how we want to position information on the new document.

**Comment by JJdeGroot** [comment]
Hi @shawna-slh thanks, any ETA when we can expect a definitive choice from you, @iadawn  and AG Chairs to implement option 1 or 2?

I know CSUN is going on now - but would be nice to communicate an ETA to our TF because our CFC keeps moving ahead in time.

**Comment by shawna-slh** [comment]
Hi @JJdeGroot Thank you for pinging so I can clarify: I shared my perspective and am fine with what the group decides. Thank you for considering my input. I leave it in AG leadership's hands.

<below are casual comments, in case they are helpful. W3C Process and AG processes and preferences prevail>

The title follows the same process as the rest of the doc, i.e.: Anyone can comment. TF develops consensus and proposes to AG. AG Chairs manage AG review and approval.

I don't know if @iadawn or AG Chairs have additional input? To move things along in case they don't (and because they are busy at CSUN), perhaps good next steps would be:
* Within the TF, check if the 3 votes for Option 1 are also OK with Option 2 if that's what the TF wants to go with.
* Submit updated info to the AG Chairs so they can get the CfC going in AG when they have a chance.

**Comment by JJdeGroot** [comment]
Following last week's decision, the title of our guidance [has been updated](https://github.com/w3c/matf/commit/a3a5035f4af6662baf610cdabd6bada571026e11) to:

**Guidance on Applying WCAG 2.2 to Mobile Applications (WCAG2Mobile)**
